cms/djangoapps/export_course_metadata/apps.py

```diff
@@ -0,0 +1,18 @@
+"""
+Define the export_course_metadata Django App.
+"""
+
+from django.apps import AppConfig
+
+
+class ExportCourseMetadataConfig(AppConfig):
+    """
+    App for exporting a subset of course metadata
+    """
+    name = u'cms.djangoapps.export_course_metadata'
+
+    def ready(self):
+        """
+        Connect signal handler that exports course metadata
+        """
+        from . import export_course_metadata  # pylint: disable=unused-import
```

Comment:
Normally we'd name this `signals.py` instead of replicating the module name. Not saying you should necessarily. But it took me a sec to figure out why we had a file with the same name as the module and how it connected to `apps.py`